service/src/main/java/com/codeforcommunity/requester/MapRequester.java

```diff
@@ -41,85 +46,132 @@ public MapRequester(Vertx vertx) {
    * Makes a request to the ArcGIS mapping service to change the status of the all of the blocks in
    * the given list to the specified new block status.
    *
-   * @param streetIds a list of block IDS.
+   * @param blockIds a list of block IDs.
    * @param updateTo The block status all blocks should be updated to.
    */
-  public void updateStreets(List<String> streetIds, BlockStatus updateTo) {
-    updateLayers(streetIds, updateTo, this.tokenFuture);
+  public void updateBlocks(List<String> blockIds, BlockStatus updateTo) {
+    updateLayers(blockIds, updateTo, this.tokenFuture);
   }
 
   /** Make a request to update the ArcGIS feature layer */
   private Future<Void> updateLayers(
```

Comment:
Let's split this into two methods, one for getting the ids and one for doing the update